libp2p/Service.cpp

```diff
@@ -1162,10 +1172,11 @@ bool Service::erasePeers(std::vector<dev::network::NodeIPEndpoint> const& endpoi
     setStaticNodes(nodes);
     SERVICE_LOG(INFO) << LOG_DESC("erase peers to the running node successfully!");
     SERVICE_LOG(INFO) << LOG_DESC("try to update the configfile now");
-    return updatePeersToIni(nodes);
+    return updatePeersToIni(nodes, response);
 }
 
-bool Service::updatePeersToIni(std::map<dev::network::NodeIPEndpoint, NodeID> const& nodes)
+bool Service::updatePeersToIni(
+    std::map<dev::network::NodeIPEndpoint, NodeID> const& nodes, std::string& response)
 {
     std::string tmpdata = "";
```

Comment:
if updatePeersToIni will throw exception, and the method call it handle this exception

Reply:
ok